recipes/_packaging.rb

```diff
@@ -58,3 +62,5 @@
   omnibus_env['PATH'] << node['wix']['home']
   omnibus_env['PATH'] << node['7-zip']['home']
 end
+
+include_recipe 'omnibus::_libffi' if ppc64? || ppc64le?
```

Comment:
yes, that's correct. Also if these libraries are missing, the embedded libffi source within extension is older version fails on ppc64, hence we need this.